src/ordering.jsx

```diff
@@ -0,0 +1,27 @@
+export const reorderObjectInIndexedArray = (
```

Comment:
I wrote this function ahead of the session since it was a little complicated. How it works:

```js
// given blog titles
const blogTitles  =  [{index: 1, title: 'Lesson 1.10'}, {index: 0, 'Lesson 1.11'}, {index: 2, 'Lesson 5.1'}]
console.log(reorderObjectInIndexedArray(1, 0, blogTitles))
  // returns  [{index: 0, title: 'Lesson 1.10'}, {index: 1, 'Lesson 1.11'}, {index: 2, 'Lesson 5.1'}]
```